pyphare/pyphare/pharesee/hierarchy.py

```diff
@@ -1076,7 +1076,7 @@ def hierarchy_from(simulator=None, qty= None, pop = "", h5_filename=None, time=N
 def merge_particles(hierarchy):
 
     for time, patch_levels in hierarchy.time_hier.items():
-        for ilvl, plvl in hierarchy.patch_levels.items():
+        for ilvl, plvl in patch_levels.items():
```

Comment:
maybe we should rather use the levels() method? (if we deprecate patch_levels soonish)